views/new-message.ejs

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
This commit is a bugfix for the route, right? Should this file be committed here?